app/assets/javascripts/admin/controllers/admin-reports.js.es6

```diff
@@ -16,15 +17,21 @@ export default Ember.Controller.extend({
     return arr.concat( Discourse.Site.currentProp('sortedCategories').map(function(i) { return {name: i.get('name'), value: i.get('id') }; }) );
   }.property(),
 
+  groupOptions: function() {
+    var arr = [{name: I18n.t('admin.dashboard.reports.groups'), value: 'all'}];
+    return arr.concat( this.site.groups.map(function(i) { return {name: i['name'], value: i['id'] }; }) );
```

Comment:
Nitpick here. The brackets and parentheses are not consistently spaced. Also...`ES6` functions` \o/ (i) =>`